schemas/bookbrainz.sql

```diff
@@ -566,6 +566,36 @@ ALTER TABLE bookbrainz.language_set__language ADD FOREIGN KEY (language_id) REFE
 ALTER TABLE bookbrainz.edition_data ADD FOREIGN KEY (language_set_id) REFERENCES bookbrainz.language_set (id);
 ALTER TABLE bookbrainz.work_data ADD FOREIGN KEY (language_set_id) REFERENCES bookbrainz.language_set (id);
 
+CREATE TABLE bookbrainz.title_type (
+	id SERIAL PRIMARY KEY,
+	title VARCHAR(40) NOT NULL CHECK (title <> ''),
```

Comment:
I was thinking it might be worth limiting it just so we always know how long it could possibly be for the design. Im happy to use text if you think that is more appropriate.